src/calendar/index.js

```diff
@@ -357,6 +363,65 @@ class Calendar extends Component {
     return CalendarHeader;
   }
 
+  showLoader() {
+    if (this.props.LoaderComponent) {
+      return <LoaderComponent />;
```

Comment:
this way, `LoaderComponent` will be undefined.
It should be `return this.props.LoaderComponent()`

thoughts?

Reply:
Please create a PR. I'll merge it

Reply:
Fork the repo, make changes to your forked repo and then you can create PR